apps/balancer-tools/src/components/SearchPoolForm/index.tsx

```diff
@@ -71,17 +74,19 @@ export function SearchPoolForm({
       poolTypeFilter?.length ? { poolTypes: poolTypeFilter } : {},
       { revalidateIfStale: true },
     );
-  const isPool = !!poolsData?.pools?.length;
 
   function handleSubmitForm(formData: PoolAttribute) {
     onSubmit?.(formData);
     close?.();
     closeCombobox();
   }
 
-  const filteredPoolList = poolsDataList?.pools.filter((pool) =>
-    filterPoolInput({ poolSearchQuery: poolId, pool }),
-  );
+  const filteredPoolList = poolsDataList?.pools
+    .filter((pool) => filterPoolInput({ poolSearchQuery: poolId, pool }))
+    .filter((pool) => filterPoolWithLiveGauge({ pool, onlyVotingGauges }));
```

Comment:
We don't want this filter everywhere. For instance, in the metadata app it's going to hide pools that must be searchable.

Reply:
yup, but this is handled inside filterPoolWithLiveGauge as shown on the video. If `onlyVotingGauges` is false, which is the default value, this filter will have no effect on `poolsDataList?.pools`